core/controllers/learner_dashboard.py

```diff
@@ -32,19 +31,12 @@ class LearnerDashboardPage(base.BaseHandler):
 
     @acl_decorators.can_access_learner_dashboard
     def get(self):
-        if self.username in config_domain.BANNED_USERNAMES.value:
```

Comment:
a) They are removed from controllers. Only place they are left is rights_manager and test_utils which would be removed in next PR.
b) After next PR, the old system should be completely replaced. So, we will turn on the update role functionality in that.